MapboxCoreNavigation/FreeDriveLocationManager.swift

```diff
@@ -0,0 +1,196 @@
+import MapboxNavigationNative
+import CoreLocation
+import Mapbox
+import MapboxAccounts
+
+func createNavigator(withTilesVersion tilesVersion: String = "2020_03_07-03_00_00") -> Navigator {
+    let settingsProfile = SettingsProfile(application: ProfileApplication.kMobile, platform: ProfilePlatform.KIOS)
+    let navigator = Navigator(profile: settingsProfile, customConfig: "")
+    let host = "https://api-routing-tiles-here-staging.tilestream.net"
+    let publicToken = ""
+    assert(publicToken != "")
+    let endpointConfig = TileEndpointConfiguration(
+        host: host,
+        version: tilesVersion,
+        token: publicToken,
+        userAgent: "MapboxNavigationNative",
+        navigatorVersion: "14.1.0", skuTokenSource: Accounts()
+    )
```

Comment:
This all needs to be cleaned up to not hard-code a staging endpoint, outdated navigation-native version, and tile version.

Reply:
sure, that's why it is a draft